File: src/main/java/ru/zakazsharovekb/airdelivery/model/Customer.java
package ru.zakazsharovekb.airdelivery.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.Embeddable;

@Getter
@Setter
@Embeddable
public class Customer {
    private String name;
    private String phone;
    private String email;
}

File: delivery_mobile_app/lib/customer.dart
class Customer {
  final String name;
  final String phone;
  final String email;

  const Customer(
      {required this.name, required this.phone, required this.email});

  factory Customer.fromJson(Map<String, dynamic> customerJson) {
    return Customer(
        name: customerJson['name'] ?? "",
        phone: customerJson['phone'] ?? "",
        email: customerJson['email'] ?? "");
  }
}
